### PR TITLE
Daos 2170

### PR DESCRIPTION
**Pull request by GitHuaKuang**

### CODE REVIEW

File: src/tests/ftest/container/attribute.py [hunk 1]
```diff
@@ -80,6 +81,7 @@ class ContainerAttributeTest(TestWithServers):
     def setUp(self):
         super(ContainerAttributeTest, self).setUp()
 
+        self.logger = logging.getLogger("ConAttributeLogger")
```
Comment (brianjmurrell) [inline src/tests/ftest/container/attribute.py]: Any reason to use a separate logger when **apricot** already has a `DaosLog` handle open?

https://github.com/daos-stack/daos/blob/db0eec3dd6b55a86b69b5d35561f6ba9d4d646d4/src/tests/ftest/util/apricot/apricot/test.py#L129

File: src/tests/ftest/pool/attribute.py [hunk 2]
```diff
@@ -82,42 +74,30 @@ def verify_get_attr(indata, outdata):
...
-class PoolAttributeTest(Test):
+class PoolAttributeTest(TestWithServers):
     """
     Test class Description: Tests DAOS pool attribute get/set/list.
+    :avocado: recursive
     """
     def setUp(self):
+        super(PoolAttributeTest, self).setUp()
+
+        self.logger = logging.getLogger("PoolAttributeLogger")
```
Comment (brianjmurrell) [inline src/tests/ftest/pool/attribute.py]: Ditto.

File: src/tests/ftest/container/attribute.py (hunk 1 above)
Comment (GitHuaKuang) [reply src/tests/ftest/container/attribute.py]: this apricot d_log is for TestWithoutServers class, somehow it's not defined in TestWithServers class that used here.
Two solutions: 
1. Add the d_log definition in TestWithServers
2. Define it locally in attributeTest classes

Suggestions?

Comment (brianjmurrell) [reply src/tests/ftest/container/attribute.py]: TestWithServers inherits from TestWithoutServers:
https://github.com/daos-stack/daos/blob/b7c562626c652e0b7a63c7c0f693bd020234ed0d/src/tests/ftest/util/apricot/apricot/test.py#L139

**Pull request merged**